kong/cmd/health.lua

```diff
@@ -0,0 +1,54 @@
+local conf_loader = require "kong.conf_loader"
+local log = require "kong.cmd.utils.log"
+local kill = require "kong.cmd.utils.kill"
+local pl_stringx = require "pl.stringx"
+local pl_path = require "pl.path"
+local pl_tablex = require "pl.tablex"
```

Comment:
Please order those (I know it sounds silly but so far all new modules are ordered and as a result among many other efforts, the code is more readable).

See: https://github.com/Mashape/kong/blob/refactor/cli/kong/cmd/stop.lua#L1-L7